src/app/api/breach-icon/[breach-domain]/route.ts

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { readFile } from "node:fs/promises";
+import { dirname, resolve as pathResolve } from "node:path";
+import { fileURLToPath } from "node:url";
+import { NextResponse } from "next/server";
+
+export async function GET(
+  request: Request,
+  context: { params: { "breach-domain": string } }
+) {
+  const logoPath = pathResolve(
+    dirname(fileURLToPath(import.meta.url)),
+    `../../../../../public/logo_cache/${context.params["breach-domain"]}.ico`
+  );
+  try {
+    const file = await readFile(logoPath);
```

Comment:
I know this is just a temporary solution, but any concerns here with a disk i/o operation on every logo request?

Reply:
I'd say yes, it feels pretty concerning to me. @mansaj is now looking at setting up the cronjob+CDN, so hopefully we won't need this PR.